### DISCUSSION TITLE
要運算快還得會說人話，美特戰司令部加速開發可解釋 AI 協助作戰

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#651] [科技新知]
近期在佛羅里達州召開的國際特種部隊工業會議中，美國特戰司令部數據長表示，將與國防高等研究計劃署合作，加速開發可解釋 AI 系統，讓系統在協助特種部隊作戰時，能符合倫理規範外，還能夠清楚解釋系統進行決策和溝通的所有過程。
近期可能因為國際情勢的關係，科技對於軍事方面的進度似乎也快了起來!
https://technews.tw/2022/06/06/ussocom-is-pushing-the-xai-technology-for-future-operations/

**Comment by leon10456** [comment]
感謝樓主的分享 這感覺超棒的 但還是希望不要有戰爭..

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by etin2468** [comment]
好多軍事單位都導入AI了，未來真的打起來好可怕

**Comment by 110409538** [comment]
感謝分享！

**Comment by Lynn12378** [comment]
好擔心科技運用在戰爭真的會帶來很大的影響

**Comment by Jimmy911218** [comment]
ai用在軍事上對未來真的是好是壞好難說阿

**Comment by Hchejxnc** [comment]
科技的發展真的非常迅速
